src/test/javascript/spec/service/faq.service.spec.ts

```diff
@@ -0,0 +1,208 @@
+import { TestBed } from '@angular/core/testing';
+import { HttpClientTestingModule, HttpTestingController } from '@angular/common/http/testing';
+import { HttpResponse } from '@angular/common/http';
+import { take } from 'rxjs/operators';
+import { ArtemisTestModule } from '../test.module';
+import { LocalStorageService, SessionStorageService } from 'ngx-webstorage';
+import { MockSyncStorage } from '../helpers/mocks/service/mock-sync-storage.service';
+import { TranslateService } from '@ngx-translate/core';
+import { MockTranslateService } from '../helpers/mocks/service/mock-translate.service';
+import { Course } from 'app/entities/course.model';
+import { FAQ, FAQState } from 'app/entities/faq.model';
+import { FAQCategory } from 'app/entities/faq-category.model';
+import { FAQService } from 'app/faq/faq.service';
+
+describe('Faq Service', () => {
+    let httpMock: HttpTestingController;
+    let service: FAQService;
+    const resourceUrl = 'api/faqs';
+    let expectedResult: any;
+    let elemDefault: FAQ;
+
+    beforeEach(() => {
+        TestBed.configureTestingModule({
+            imports: [ArtemisTestModule, HttpClientTestingModule],
+            providers: [
+                { provide: LocalStorageService, useClass: MockSyncStorage },
+                { provide: SessionStorageService, useClass: MockSyncStorage },
+                { provide: TranslateService, useClass: MockTranslateService },
+            ],
+        });
+        service = TestBed.inject(FAQService);
+        httpMock = TestBed.inject(HttpTestingController);
+
+        expectedResult = {} as HttpResponse<FAQ>;
+        elemDefault = new FAQ();
+        elemDefault.questionTitle = 'Title';
+        elemDefault.course = new Course();
+        elemDefault.questionAnswer = 'Answer';
+        elemDefault.id = 1;
+        elemDefault.faqState = FAQState.ACCEPTED;
+    });
+
+    afterEach(() => {
+        httpMock.verify();
+    });
+
+    describe('Service methods', () => {
+        it('should create a faq in the database', async () => {
```

Comment:
I agree with CodeRabbit here. Unless you're `await`ing something in these methods, you can remove the `async` keyword.

More of a nitpick: The client doesn't directly interact with the database. I'd just leave out the "in the database" in these test case names

Reply:
Addressed